packages/react/src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -70,16 +70,31 @@ export const ProgressStep = ({ ...props }) => {
     );
   };
 
+  let message = 'Incomplete';
```

Comment:
g11n; Do we want to add a prop to let app specify the assistive text? (Though translate-ability for assistive text didn't use to be a hard-requirements, I have seen several issues asking for it)

Reply:
Good idea! I added it, it's the first time I've done it so let me know if it all looks good!